docs/style.css

```diff
@@ -16,893 +16,76 @@ You should have received a copy of the GNU General Public License
-	margin: auto;
-}
-
-div#cgit div.content table {
-	font-size: small;
-}
-
-div#cgit div.content {
-	margin-top: 1em;
-}
-
-div#cgit span.deletions {
-	color: #800;
-}
-
-div#cgit span.insertions {
-	color: #080;
-}
-
-div#cgit table.diff td div.add {
-	color: green;
 }
 
-div#cgit table.diff td div.del {
-	color: red;
-}
-
-div#cgit table.tabs {
-	border: 1px solid black;
-	border-top: 0;
-}
-
-div#cgit table.tabs input {
-	width: 10em;
-	height: 100%;
-	font-size: small;
-}
-
-div#cgit table.tabs input[type="submit"] {
-	height: 100%;
-	width: 5em;
-}
-
-div#cgit table.tabs form {
-	display: flex;
-	flex-direction: row;
-	justify-content: flex-end;
-}
-
-div#cgit a.branch-deco {
-	color: #000;
-	margin: 0px 0.5em;
-	padding: 0px 0.25em;
-	background-color: #88ff88;
-	border: solid 1px #007700;
-}
-
-div#cgit a.deco {
-	color: #000;
-	margin: 0px 0.5em;
-	padding: 0px 0.25em;
-	background-color: #ff8888;
-	border: solid 1px #770000;
-}
-
-div#cgit table.tabs a {
-	font-size: 100%;
-	margin-left: 1em;
-}
-
-div#cgit table.tabs a.active {
-	background-color: lightgrey;
-}
-
-
-div#cgit table.tabs td {
-	padding: 0;
-	width: 40%;
-	border: 0;
-	height: 100%;
-}
-
-div#cgit table.tabs td:nth-child(2) {
-	width: 50%;
-}
-
-div#cgit table.tabs tr { 
-	height: 100%;
-	display: flex;
-	flex-direction: row;
-	justify-content: space-between;
-	vertical-align: middle;
-	align-items: center;
-}
-
-div#cgit div.footer {
-	margin-top: 0.5em;
-	text-align: center;
-	font-size: 80%;
-	color: #ccc;
-}
-
-div#cgit td form {
-	text-align: center;
-}
-
-div#cgit td form input,
-div#cgit td form select,
-div#cgit td form button {
-	margin: 3px;
-}
-
-div#cgit {
-	width: 100%;
-}
-
-div#cgit table#header tr {
-	display: flex;
-	flex-direction: row;
-	align-content: flex-start;
-	align-items: center;
-	width: 100%;
-	height: 8.1em;
-}
-
-div#cgit table#header tr:nth-child(2) {
-	padding-top: 10px 0;
-	border-top: 1px solid black;
-	vertical-align: baseline;
-	height: 3em;
-}
-
-div#cgit h1,
-div#cgit td.main {
-	display: block;
-	font-size: 2em;
-	font-weight: bold;
-}
-
-div#cgit table {
-	border-collapse: collapse;
-}
-
-div#cgit table#header td.logo {
-	margin: 0;
-	padding: 0;
-	width: 19%;
-	height: 100%;
-	border: 0;
-}
-
-div#cgit table#header td.main {
-	font-size: 1.5em;
-	flex-flow: column;
-	width: 100%;
-	border: 0;
-}
-
-div#cgit table#header td.form {
-	font-size: 1.5em;
-	height: 90%;
-	display: flex;
-	flex-flow: column;
-	width: 20%;
-	justify-content: center;
-	border: 0;
-}
-
-div#cgit table#header td.form select {
-	height: 1.5em;
-	width: 6em;
-}
-
-div#cgit table#header td.form input {
-	height: 1.5;
-	width: 6em;
-}
-
-div#cgit table#header td.sub {
-	display: flex;
-	flex-flow: column;
-	width: 100%;
-	border: 0;
-}
-
-div#cgit table#header td.sub.right {
-	flex-flow: column;
-	width: 10%;
-	font-style: italic;
-	color: #555555;
-}
-
-div#cgit ul.pager {
-	display: flex;
-	justify-content: center;
-	list-style: none;
-	text-align: center;
-}
```

Comment:
I think we want to do something subtle with this: 

<img width="495" alt="Screenshot 2024-01-25 at 23 13 08" src="https://github.com/underground-software/orbit/assets/1909841/6a032425-a511-4550-8c26-fb88a7453921">